### PR TITLE
hooks? i guess

### PR DESCRIPTION
**Pull request by elmouster**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-scripts": "5.0.0",
+    "react-transition-group": "^4.4.2",
```
Comment (minbelapps) [inline package.json:12]: You don't need this

File: src/radio-widget/components/radio-station-list/radio-station-list.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.station-list {
+    height: 340px;
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+    background-color: darkslategrey;
+}
```
Comment (elmouster) [inline src/radio-widget/components/radio-station-list/radio-station-list.css]:
```suggestion
}

```

File: src/radio-widget/components/radio-station/radio-station.css [hunk 3]
```diff
@@ -1,7 +1,56 @@
...
+    flex-direction: row;
+    justify-content: space-between;
+    color: white;
+    opacity: 0.66;
+}
+
+.station-picture {
+    border-radius : 50%;
+    border: 2px solid white;
 }
```
Comment (elmouster) [inline src/radio-widget/components/radio-station/radio-station.css]:
```suggestion
}

```

File: src/radio-widget/components/radio-station-list/radio-station-list.js [hunk 4]
```diff
@@ -9,8 +10,9 @@ const RadioStationList = ({stations, onRadioSelect, currentlyPlayingIndex}) => {
                 onRadioSelect={onRadioSelect} 
                 title={station.title}
                 wave={station.wave}
+                pictureUrl={station.pictureUrl}
             />
         </li>);
-    return (<ul>{stationListItems}</ul>)
+    return (<ul class="station-list">{stationListItems}</ul>)
 }
 export default RadioStationList;
```
Comment (elmouster) [inline src/radio-widget/components/radio-station-list/radio-station-list.js]:
```suggestion
export default RadioStationList;

```

File: src/radio-widget/containers/radio-station-list/radio-station-widget.css [hunk 5]
```diff
@@ -13,4 +50,24 @@
...
+    opacity: 1;
+}
+
+.expand-button {
+    padding-left: 12px;
+}
+
+.on-of-button {
+    padding-right: 12px;
 }
```
Comment (elmouster) [inline src/radio-widget/containers/radio-station-list/radio-station-widget.css]:
```suggestion
}

```

File: src/radio-widget/containers/radio-station-list/radio-station-widget.js [hunk 6]
```diff
@@ -3,12 +3,11 @@ import RadioStationList from '../../components/radio-station-list/radio-station-
 import { fetchStations } from '../../services/radio-station-service.js'
 import './radio-station-widget.css';
 
-const RadioStationWidget = ({currentlyPlaying, currentlyPlayingIndex}) => {
-    const [state, setState] = useState({currentlyPlaying, currentlyPlayingIndex});
+const RadioStationWidget = ({currentlyPlayingIndex}) => {
+    const [state, setState] = useState({currentlyPlayingIndex});
```
Comment (minbelapps) [inline src/radio-widget/containers/radio-station-list/radio-station-widget.js:7]: const [currentlyPlayingIndex, setCurrentlyPlayingIndex] =

**Review by minbelapps — APPROVED** [review]

**Pull request merged**